lib/sycamore/sycamore/utils/fileformat_tools.py

```diff
@@ -0,0 +1,44 @@
+import subprocess
+from tempfile import NamedTemporaryFile
+
+from sycamore.data import Document
+
+
+def convert_file_to_pdf(doc: Document) -> Document:
+    """
+    Utility to convert binary_representations into different file formats. Uses LibreOffice as the conversion engine.
+
+    Note: LibreOffice currently requires manual installation based on your platform.
+    """
+
+    def run_libreoffice(source_path, output_path):
+        subprocess.run(
+            [
+                "libreoffice",
+                "--headless",
+                "--convert-to",
+                "pdf",
+                source_path,
+                "--outdir",
+                output_path,
+            ]
+        )
+
+    assert doc.binary_representation is not None
+
+    current_filetype = doc.properties.get("filetype")
```

Comment:
.get("filetype", "unknown")

and remove the assertion. Unless libreoffice is using the extention for file type info

Reply:
it does from what I see so gonna leave this

Reply:
actually its a little unclear, their docs aren't very clear. Still useful for us to keep so we aren't supporting magic types

Reply:
ok. If it's unclear what libreoffice will do it's safer to do the clearly correct thing.

Reply:
Which is enforcing/being aware of the extension I'd think?

Reply:
Per discussion, get the extension from the original filesystem.